### PR TITLE
Deleted one of the servos in the climber subsystem

### LINKED ISSUE #29: Remove Second Servo
The two hood servos are tied together physically, and any inputs are shared between them. One of the objects in the code is currently useless, please remove the unused `Servo` object.

### PR DESCRIPTION
**Pull request by rakshaaJey**
Fix #29

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Shooter.java [hunk 1]
```diff
@@ -52,8 +52,7 @@ public Shooter() {
     m_left.burnFlash();
     m_right.burnFlash();
 
-    m_leftHood = new Servo(RobotMap.PWM.LEFT_HOOD_SERVO);
-    m_rightHood = new Servo(RobotMap.PWM.RIGHT_HOOD_SERVO);
+    m_hoodServo = new Servo(RobotMap.PWM.HOOD_SERVO);
     m_pot = new AnalogInput(RobotMap.AIO.HOOD_POT);
 
     m_rightHood.setBounds(2.5, 2.45, 1.5, 0.55, 0.5);
```
Comment (Jacob4649) [inline src/main/java/frc/robot/subsystems/Shooter.java]: replace `m_rightHood` with `m_hoodServo` here

Comment (Jacob4649) [reply src/main/java/frc/robot/subsystems/Shooter.java]: > replace `m_rightHood` with `m_hoodServo` here

**Comment by rakshaaJey** [comment]
Okay, changed it to m_hoodServo 
b198b51

**Comment by matthew-reynolds** [comment]
Oops, didn't mean to delete that. Jacob asked for `m_rightHood` to be renamed `m_hoodServo`

**Review by Jacob4649 — APPROVED** [review]

**Pull request merged**